src/main/java/com/vmware/vipclient/i18n/VIPCfg.java

```diff
@@ -319,6 +312,7 @@ public String getVipServer() {
 
     public void setVipServer(String vipServer) {
         this.vipServer = vipServer;
+        this.addMsgOriginsQueue(DataSourceEnum.VIP);
```

Comment:
Because service takes priority over offline bundles, so we should make sure service will be before offline bundles.

Reply:
We make sure of this sequence (VIP first before offline) when VIPCfg.initialize(configfile) is used. However, if product team chooses to change the priority, they should be able to do so by calling the individual setters in the order they prefer.